tests/codegen/issues/issue-112169-long-iterator-unroll.rs

```diff
@@ -0,0 +1,23 @@
+//@ compile-flags: -O
+#![crate_type = "lib"]
+
+// Test that simple iterator-based loops of length > 101 are fully optimized away.
+// See <https://github.com/rust-lang/rust/issues/112169>
+
+// CHECK-LABEL: issue_112169()
+#[no_mangle]
+pub fn issue_112169() -> i32 {
+    // CHECK-NEXT: {{.*}}:
```

Comment:
I think we should be able to get rid of this line, can you try without it? Comparing to https://github.com/rust-lang/rust/blob/master/tests/codegen/issues/issue-101082.rs, that compiles to similar LLVM IR but doesn't seem to need this in the test case.

```
define noundef i32 @issue_112169() unnamed_addr {
bb4.peel.next:
  ret i32 102
}

define noundef i64 @test() unnamed_addr personality ptr @rust_eh_personality {
bb5:
  ret i64 165
}

declare noundef i32 @rust_eh_personality(i32 noundef, i32 noundef, i64 noundef, ptr noundef, ptr noundef) unnamed_addr #2
```

Reply:
Ah I see this was discussed above, never mind.